### ISSUE TITLE
Hung download of maxresdefault.jpg drains battery

### ISSUE THREAD

**Issue by mikefreemen** [lbryio/lbry-android#821]
## The Issue
When I launch the app, I see a notification in the system pull down as follows that shows downloading maxresdefault.jpg and it never advances beyond 28%.  Further, I've now confirmed that whatever it's doing instead of downloading the image, is draining my battery at quite an impressive rate.  In airplane mode, over 8ish hours while I slept, my battery was drained from 93% to 0%.  If i haven't launched the LBRY app, I would never lose more than like 10% max during the night, and in airplane mode, I think it's actually more like 3%.

### Steps to reproduce
1. Launch LBRY app (on Moto X4, Android v9)
2. see the picture message in the notifications list
3. put in airplane mode
4. watch battery drain.

### Expected behaviour
This download should not get stuck at 28%.  Even if it does, it should not kill my battery.

### Actual behaviour
Download notification stays in the notifications unchanged until I reboot the phone (days at a time if plugged in while I sleep).  A failed download should never kill a battery.

## System Configuration
- Device: Moto X4
- Android version: 9

## Anything Else
<Nothing>

## Screenshots
![Screenshot_20200121-205205](https://user-images.githubusercontent.com/1026353/72859781-f4500c80-3c92-11ea-88d4-02d275851e53.png)

### Acceptance Criteria
1.
2.
3.

### Definition of Done
- [ ]  Tested against acceptance criteria
- [ ] Tested against the assumptions of the user story
- [ ] The project builds without errors
- [ ] Unit tests are written and passing
- [ ] Tests on devices/browsers listed in the issue have passed
- [ ] QA performed & issues resolved
- [ ] Refactoring completed
- [ ] Any configuration or build changes documented
- [ ] Documentation updated
- [ ] Peer Code Review performed

**Comment by tzarebczan** [comment]
Sorry to hear about the trouble. Are you able to see this in the Library section and delete it?

The downloads should typically time out, so not sure why they would keep running. Let us know the URL too so we can give it a shot here

**Comment by mikefreemen** [comment]
Ah, I didn't know I could download stuff.  Not even sure how I requested the app do that. :) URL is 
https://open.lbry.com/@MemeKing:9/tomsafejulie:6

So I went into the Library menu item, long-pressed to select this video (only video in the list), then selected the trash can icon to delete it.  Now the list is empty.  I switched to app switcher and closed the app (swiped it up to close the instance).  At this point, the app appears to not be running.  I then go into the Notifications, and the same message is there, but it now says 0%.  So I believe the problem persists.  

Finally, I rebooted to see if the problem is gone after deleting the item to download.  Interesting enough, a different image "maxresdownload_2.jpg" showed in the notifications as download complete, showed up in my Library section.  I then deleted it, and I hope this is all done for now.  I'm not going to reboot again.  At this point, I think my battery will not drain...

Let me know if I can provide any other details to help diagnose the problem.
